packages/common/base/src/blockchain/types/evm.ts

```diff
@@ -10,6 +10,7 @@ export const EVMBlockchain = {
     ZORA: "zora",
     ARBITRUMNOVA: "arbitrumnova",
     ZKATANA: "zkatana",
+    ZKYOTO: "zkyoto",
```

Comment:
we should remove ZKATANA to be consistent with the other changes

Reply:
I will leave it to handle it's removal in another PR, thanks for the catch!!